### PR TITLE
Adds description for JUnit 5 usage

### PR DESCRIPTION
**Pull request by artjomka**
Adds separate section in usage documentation to address differences in JUnit4/JUnit5 approaches.

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -119,3 +119,34 @@ ambassador.container.image=replacement image name here
...
+
+```java
+public class SimpleMySQLTest {
+    @Rule
+    public MySQLContainer mysql = new MySQLContainer();
+```
+
+### JUnit 5
+
+**JUnit5**: you need to manually start container in *@BeforeAll* annotated method in test (tear down happens automatically on test end)
```
Comment (bsideup) [inline docs/usage.md]: this example also does not reflect the previous one, with JUnit 4. Maybe we should replace it with `@BeforeEach`/`@AfterEach`

File: docs/usage.md [hunk 2]
```diff
@@ -119,3 +119,34 @@ ambassador.container.image=replacement image name here
 vncrecorder.container.image=replacement image name here
 tinyimage.container.image=replacement image name here
 ```
+
+## JUnit
+
+### Junit 4
+ 
+**JUnit4 @Rule/@ClassRule**: this mode starts container before your tests and tears it down afterwards.
```
Comment (kiview) [inline docs/usage.md]: `this` with uppercase

File: docs/usage.md [hunk 3]
```diff
@@ -119,3 +119,34 @@ ambassador.container.image=replacement image name here
...
+public class SimpleMySQLTest {
+    @Rule
+    public MySQLContainer mysql = new MySQLContainer();
+```
+
+### JUnit 5
+
+**JUnit5**: you need to manually start container in *@BeforeAll* annotated method in test (tear down happens automatically on test end)
+
+Start container in @BeforeAll method:
```
Comment (kiview) [inline docs/usage.md]: `Start container in a method annotated with @BeforeAll:`

File: docs/usage.md (hunk 1 above)
Comment (kiview) [reply docs/usage.md]: Or alternatively, use `@ClassRule` in the JUnit4 example.

Comment (kiview) [reply docs/usage.md]: Also please uppercase `You` and `.` at the end of the sentence.
`You need to manually start the container in a *@BeforeAll* annotated method in your test`

Comment (kiview) [reply docs/usage.md]: Also the tear down happen on JVM exit.

Comment (artjomka) [reply docs/usage.md]: Agree, will adjust it

**Comment by kiview** [comment]
@artjomka Thanks for your contribution.
I've merge the PR and will some tiny changes for grammar in a follow up commit.

**Pull request merged**